<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="TYSS-MF-033" name="com.techm.newagedelivery.api.tests.APITestSuite" tests="6" failures="1" timestamp="27 Dec 2017 15:35:04 GMT" time="22.304" errors="0">
  <testcase name="getJenkinsJobHistory" time="22.304" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <failure type="java.lang.AssertionError" message="Status code is not equal to 200 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Status code is not equal to 200 expected [200] but found [404]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:652)at com.techm.newagedelivery.api.factory.BaseApiLib.verifyStatus(BaseApiLib.java:25)at com.techm.newagedelivery.api.tests.APITestSuite.getJenkinsJobHistory(APITestSuite.java:178)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </failure>
  </testcase> <!-- getJenkinsJobHistory -->
  <testcase name="listBranch" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <skipped/>
  </testcase> <!-- listBranch -->
  <testcase name="createSubTask" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <skipped/>
  </testcase> <!-- createSubTask -->
  <testcase name="getProjectInfo" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <skipped/>
  </testcase> <!-- getProjectInfo -->
  <testcase name="createIssue" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <skipped/>
  </testcase> <!-- createIssue -->
  <testcase name="changeIssueStatus" classname="com.techm.newagedelivery.api.tests.APITestSuite">
    <skipped/>
  </testcase> <!-- changeIssueStatus -->
</testsuite> <!-- com.techm.newagedelivery.api.tests.APITestSuite -->
